sda/cmd/s3inbox/proxy.go

```diff
@@ -225,6 +225,13 @@ func (p *Proxy) allowedResponse(w http.ResponseWriter, r *http.Request, token jw
 			log.Debugf("resuming work on file with fileId: %v", p.fileIds[r.URL.Path])
 		}
 
+		if err := p.storeObjectSizeInDB(rawFilepath, p.fileIds[r.URL.Path]); err != nil {
+			log.Errorf("storeObjectSizeInDB failed because: %s", err.Error())
```

Comment:
Might be better to print more info to help debugging when failed.
```suggestion
			log.Errorf("storeObjectSizeInDB failed for path '%s' (file ID: '%s') because: %s", r.URL.Path, p.fileIds[r.URL.Path], err.Error())
```

Reply:
Not here, that extra information is irrelevant.